File: src/dispatcher.js
#!/usr/bin/env node

var amqp = require('amqplib/callback_api');

amqp.connect('amqp://localhost', function(err, conn) {
  conn.createChannel(function(err, ch) {
    var q = 'task_queue';
    
    for (var x=0;x<100;x++){
    var msg = JSON.stringify({q})
        ch.assertQueue(q, {durable: true});
    ch.sendToQueue(q, new Buffer.from(msg), {persistent: true});
    console.log(" [x] Sent '%s'", msg);
    }
  });

  setTimeout(function() { conn.close(); process.exit(0) }, 500);
});

File: src/dispatcher.py
#!/usr/bin/env python
from util.sender import Sender
from util.logger import logger
import json
send=Sender('localhost','task_queue')
task={'_id':12345,'type':'plugin','ip':'173.181.212.179','port':502,'plugin':'modbus'}
print task
for i in range(2):
    task['_id']=i
    msg=json.dumps(task)
    logger.info(" Sent %r" % msg)
    send.send_msg(msg)
send.close()